packages/api-client/src/main/auth/AuthAPI.ts

```diff
@@ -109,6 +111,15 @@ class AuthAPI {
 
     return this.client.sendJSON(config).then((response: AxiosResponse) => response.data);
   }
+
+  public async headInitiateLogin(ssoCode: string): Promise<void> {
+    const config: AxiosRequestConfig = {
+      method: 'head',
```

Comment:
Usually, we use uppercase for HTTP method names

Reply:
Doch :P